compliance_checker/ioos.py

```diff
@@ -888,89 +889,163 @@ def check_platform_vocabulary(self, ds):
         val = bool(validators.url(pvocab))
         return Result(BaseCheck.MEDIUM, val, "platform_vocabulary", [m])
             
-    def _check_var_gts_ingest(self, ds, var, do_ingest, msg):
+    def _check_gts_ingest_val(self, val):
         """
-        Helper function for check_gts_ingest(). Check that a given variable
+        Check that `val` is a str and is equal to "true" or "false"
+
+        Parmeters
+        ---------
+        val (?): value to check
+
+        Returns
+        -------
+        bool
+        """
+
+        r = True
+        if isinstance(val, str):
+            if not (val.lower()=="true" or val.lower()=="false"):
+                r = False
+
+        elif (not isinstance(val, str)):
```

Comment:
This `isinstance` check is redundant given the preceding on in the `if` clause.

You could probably shorten this whole method by doing:

`return isinstance(val, str) and val.lower() in {"true", "false"}`